components/tensorflow-notebook-image/Dockerfile

```diff
@@ -126,23 +119,33 @@ RUN conda create -n py2 python=2 && \
       pip install --no-cache-dir tensorflow-model-analysis && \
       jupyter nbextension install --py --symlink tensorflow_model_analysis --user && \
       jupyter nbextension enable --py tensorflow_model_analysis --user; \
-    fi
-
-# Install jupyterlab-manager
-RUN conda install --quiet --yes \
+    fi \
+    && \
+    # Install jupyterlab-manager
+    conda install --quiet --yes \
     # nodejs required for jupyterlab-manager
     nodejs && \
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+    jupyter labextension install @jupyter-widgets/jupyterlab-manager && \
+    # Do chown in this layer for significant size savings
+    chown -R ${NB_USER}:users $HOME && \
+    chown -R ${NB_USER}:users $CONDA_DIR
 
 # Install common packages from requirements.txt for both python2 and python3
+# NB: the COPY chown can't expand a bash variable for NB_USER
 COPY --chown=jovyan:users requirements.txt $HOME/requirements.txt
```

Comment:
Why use $HOME here as opposed to some directory in /opt?

Reply:
See below.